llvm/test/CodeGen/AArch64/sched-movprfx.ll

```diff
@@ -14,13 +14,14 @@ define <vscale x 2 x i64> @and_i64_zero(<vscale x 2 x i1> %pg, <vscale x 2 x i64
 ; CHECK-NEXT:    ptrue p1.d
 ; CHECK-NEXT:    movprfx z0, z2
 ; CHECK-NEXT:    abs z0.d, p1/m, z2.d
+; CHECK-NEXT:    sel z1.d, p0, z1.d, z2.d
 ; CHECK-NEXT:    add z0.d, z0.d, z1.d
 ; CHECK-NEXT:    ret
   %data0 = tail call <vscale x 2 x i64> @llvm.abs.nxv2i64(<vscale x 2 x i64> %c, i1 0)
   %data1 = call <vscale x 2 x i64> @llvm.masked.load.nxv2i64(ptr %base,
                                                             i32 1,
                                                             <vscale x 2 x i1> %pg,
-                                                            <vscale x 2 x i64> undef)
+                                                            <vscale x 2 x i64> %c)
```

Comment:
This is a silly change to keep `%c` / `$z2` alive and ensure the `movprfx` is still generated.